tests/tracer/test_logging.py

```diff
@@ -0,0 +1,518 @@
+import os
+import re
+
+import pytest
+
+from ddtrace.internal.compat import PY2
+
+
+LOG_PATTERN = r"^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3} \w{1,} \[\S{1,}\] \[\w{1,}.\w{2}:\d{1,}\] - .{1,}$"
+
+
+@pytest.mark.parametrize("dd_trace_debug", ["true", "false", None])
+@pytest.mark.parametrize("dd_trace_log_file_level", ["CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG", None])
+@pytest.mark.parametrize("dd_trace_log_file", ["example.log", None])
+def test_unrelated_logger_loaded_first_in_debug(
+    dd_trace_debug, dd_trace_log_file_level, dd_trace_log_file, run_python_code_in_subprocess, tmpdir
+):
+    """
```

Comment:
The doc string implies this should only test `dd_trace_debug=true`. Should we remove the other parameters?

Reply:
I'm going to mark this as resolved since we went over this yesterday and refactored some of these tests. (ie: Instead of `test_unrelated_logger_loaded_first_in_debug` ,now it's just `test_unrelated_logger_loaded_first` (to test what happens to the loggers (both ddtrace and a custom logger) if another logger is configured first.